src/tests/java/picard/illumina/CheckIlluminaDirectoryTest.java

```diff
@@ -51,28 +51,30 @@ private void setUp() throws Exception {
         illuminaDir = IOUtil.createTempDir("ift_test", "IlluminaDir");
 
         interopDir = new File(illuminaDir, "InterOp");
-        if (!interopDir.mkdir()) {
+        if (!interopDir.exists() && !interopDir.mkdir()) {
```

Comment:
yes they do and since we are using parellell execution of tests now it is important to check. Alternatively we could single thread test like we currently do